components/automate-ui/src/app/page-components/simple-line-graph/simple-line-graph.component.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { SimpleLineGraphComponent } from './simple-line-graph.component';
+import * as d3 from 'd3';
+
+describe('OverviewTrendComponent', () => {
+  let fixture: ComponentFixture<SimpleLineGraphComponent>;
+  let component: SimpleLineGraphComponent;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+      ],
+      declarations: [
+        SimpleLineGraphComponent
+      ],
+      providers: [
+      ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    });
+
+    fixture = TestBed.createComponent(SimpleLineGraphComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('initializes', () => {
+    expect(component).not.toBeNull();
+  });
+
+  it('contains an svg area for the chart', () => {
+    expect(d3.select(component.svg)).not.toBeNull();
+  });
```

Comment:
I really wanted to test the items getting appended to the graph, but I was having a difficult time selecting anything within the svg or outside of it that was being appended via d3.  Any thoughts might be helpful.  I was attempting to `queryAll` using `By.css`